### ISSUE TITLE
Issue installing

### ISSUE THREAD

**Issue by Darkle** [Jack-Works/ttypescript-browser-like-import-transformer#32]
Hi, I'm trying to install ttypescript-browser-like-import-transformer, but I get the following error:
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: roffline@0.1.0
npm ERR! Found: ts-node@10.2.1
npm ERR! node_modules/ts-node
npm ERR!   dev ts-node@"^10.2.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer ts-node@"^8.0.2" from @magic-works/ttypescript-browser-like-import-transformer@3.0.0
npm ERR! node_modules/@magic-works/ttypescript-browser-like-import-transformer
npm ERR!   dev @magic-works/ttypescript-browser-like-import-transformer@"^3.0.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/coop/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/coop/.npm/_logs/2021-10-10T05_15_55_435Z-debug.log
```

**Comment by Jack-Works** [comment]
Are you using npm7?

**Comment by Darkle** [comment]
Yeah, npm version 7.21.1
I have ts-node installed as well, version 10.2.1

**Comment by Jack-Works** [comment]
Ok I think I don't really need to have ts-node as my peer dependency, therefore I will remove those dependencies later.

**Comment by Darkle** [comment]
Thanks

**Comment by Jack-Works** [comment]
hi I have published 3.0.1. Please try.

**Comment by Darkle** [comment]
I can confirm this update fixes the issue.
Thanks. 👍
